app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -15,8 +15,7 @@ import org.p2p.wallet.auth.AuthModule
 import org.p2p.wallet.common.analytics.AnalyticsModule
```

Comment:
tl;tr
- moved classes to corresponding packages
- removed unnecessary inheritance
- renamed classes to match their responsibilty